ui/core/components/stat_weights_action.ts

```diff
@@ -591,6 +675,20 @@ class EpWeightsMenu extends BaseModal {
 			<td class="current-ep"></td>
 		`;
 
+		if (this.isEpStat(stat)) {
+			const includeToggleCell = row.querySelector('.swcalc-include-toggle') as HTMLElement;
+			const cb = document.createElement('input');
+			cb.type = 'checkbox';
+			if (stat.isStat() && this.epReferenceStat == stat.getStat()) {
+				cb.checked = true;
+				cb.disabled = true;
+			} else {
+				cb.checked = statIsCalculated;
+				cb.addEventListener('change', () => this.setStatExcluded(stat, !cb.checked));
+			}
```

Comment:
Rather than use an out-of-the-box checkbox could we use a `BooleanPicker` component for this?